grml2usb

```diff
@@ -310,12 +310,13 @@ def get_defaults_file(iso_mount, flavour, name):
     bootloader_dirs = ['/boot/isolinux/', '/boot/syslinux/']
     for directory in bootloader_dirs:
         for name in name, "%s_%s" % (get_flavour_filename(flavour), name):
+            print('looking for', iso_mount + directory + name)
```

Comment:
I assume we should use `logging.info` or `logging.debug` here?

Reply:
I guess it should just go away actually. I needed this while building the "minimally required" files, but it's not useful without also looking at the code at the same time.